docs/terraformoptions.adoc

```diff
@@ -432,6 +432,11 @@ tags = {
 |
 |
 
+|check_node_active
+|Setting this variable to *one* will wait till any one worker node is active.Setting to *all* will wait till all worker node is active.*NONE* means no check.
+|"one","all","NONE"
```

Comment:
Make possible values standard i.e. all in caps or without caps. Verify what we've done where we've used none before and keep it the same please.